web/app/scripts/state/recordstate-service.js

```diff
@@ -22,6 +22,7 @@
         svc.getOptions = getOptions;
         svc.setSelected = setSelected;
         svc.getSelected = getSelected;
```

Comment:
I don't think you made this change but it looks like the previous line is repeated twice?